### PR TITLE
DFR-461 - Integrate Fortify

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
Integrate fortify into nightly pipeline

JIRA: https://tools.hmcts.net/jira/browse/DFR-461

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ruban72 — APPROVED** [review]

File: test/java/build.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+plugins {
+    id 'application'
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```
Comment (rishikrsharma) [inline test/java/build.gradle:7]: might want to make 11

File: test/java/build.gradle [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    maven {
+        url 'https://jitpack.io'
+    }
+}
+// end::repositories[]
+
+// tag::dependencies[]
+dependencies {
+    testCompile group: 'org.slf4j', name: 'slf4j-simple', version: '1.6.4'
+    testCompile 'com.github.hmcts:fortify-client:1.2.0:all'
```
Comment (rishikrsharma) [inline test/java/build.gradle:22]: if you can use 1.2.2 version, use 1.2.2

File: test/java/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-bin.zip
```
Comment (rishikrsharma) [inline test/java/gradle/wrapper/gradle-wrapper.properties:3]: might consider gradle 7.2

**Review by rishikrsharma — APPROVED** [review]
nice

**Pull request merged**